peps/pep-0767.rst

```diff
@@ -0,0 +1,548 @@
+PEP: 767
+Title: Annotating Read-Only Attributes
+Author: Eneg <eneg at discuss.python.org>
+Sponsor: Carl Meyer <carl@oddbird.net>
+Discussions-To: <REQUIRED: URL of current canonical discussion thread>
+Status: Draft
+Type: Standards Track
+Topic: Typing
+Created: 18-Nov-2024
+Python-Version: 3.14
+
+
+Abstract
+========
+
+:pep:`705` introduced the :external+py3.13:data:`typing.ReadOnly` type qualifier
+to allow defining read-only :class:`typing.TypedDict` items.
+
+This PEP proposes expanding the scope of ``ReadOnly`` to class and protocol
+attributes, as a single concise way to mark them read-only. Some parity changes
+are also made to :data:`typing.Final`.
+
+Akin to the :pep:`705`, it makes no changes to setting attributes at runtime. Correct
```

Comment:
```suggestion
Akin to :pep:`705`, it makes no changes to setting attributes at runtime. Correct
```

Reply:
English hard 😄
I thought "the" is correct here given the PEP is already mentioned two paragraphs above

Reply:
It would be correct to say "Akin to the PEP" if the referent of "the PEP" were clear in context without any disambiguation, but that is not the case here, despite it being previously mentioned. ("The PEP" could easily mean the current PEP.)

If we are saying "PEP 705" (which is what the Sphinx role will expand to), then we don't refer to individual PEPs with the definite article; we say "PEP 705", not "the PEP 705"; whether it's been referred to previously doesn't change this.